dhis-2/dhis-api/src/main/java/org/hisp/dhis/minmax/MinMaxDataElement.java

```diff
@@ -199,4 +216,17 @@ public void setGenerated( boolean generated )
     {
         this.generated = generated;
     }
+
+    public void mergeWith( MinMaxDataElement other )
+    {
+        if ( other.getClass().isInstance( this ) )
```

Comment:
You don't need this, only MinMaxDataElements are accepted in.. so should be same type already